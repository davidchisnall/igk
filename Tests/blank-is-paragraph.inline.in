
A paragraph \em{with some markup}
of text.

Another paragraph.
This one has \it{two} sentences.

Some kinds of capabilities can also be \keyword{revoked}.
This is traditionally the hardest operation to perform on capabilities.

