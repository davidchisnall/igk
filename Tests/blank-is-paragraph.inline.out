\p{
A paragraph \em{with some markup}
of text.}\p{Another paragraph.
This one has \it{two} sentences.}\p{Some kinds of capabilities can also be \keyword{revoked}.
This is traditionally the hardest operation to perform on capabilities.}